python/cuda_cccl/cuda/cccl/parallel/experimental/_bindings.pyi

```diff
@@ -43,10 +44,57 @@ class Enumeration_CCCLType:
     def STORAGE(self) -> IntEnumerationMember: ...
 
 class Enumeration_OpKind:
-    @property
-    def STATELESS(self) -> IntEnumerationMember: ...
-    @property
-    def STATEFUL(self) -> IntEnumerationMember: ...
+    STATELESS = IntEnumerationMember
+    STATEFUL = IntEnumerationMember
+    # Well-known operations
+    PLUS: IntEnumerationMember
+    MINUS: IntEnumerationMember
```

Comment:
Why do we assign STATELESS and STATEFUL but use type annotations for the others?